backend/build.gradle.kts

```diff
@@ -276,6 +278,19 @@ tasks.named<Checkstyle>("checkstyleTest") {
     configFile = rootProject.file("checkstyle/config-test.xml")
 }
 
+tasks.named<DependencyUpdatesTask>("dependencyUpdates").configure {
+    rejectVersionIf {
+        candidate.group == "io.sentry" || (isNonStable(candidate.version) && !isNonStable(currentVersion))
+    }
+}
+
+fun isNonStable(version: String): Boolean {
+    val stableKeyword = listOf("RELEASE", "FINAL", "GA").any { version.uppercase().contains(it) }
+    val regex = "^[0-9,.v-]+(-r)?$".toRegex()
+    val isStable = stableKeyword || regex.matches(version)
+    return isStable.not()
+}
```

Comment:
this function is now basically twice in this file (line 150 `isStable`), can we reuse one of them?

Reply:
Jep good catch!